marklogic-data-hub/src/test/java/com/marklogic/hub/EndToEndFlowTests.java

```diff
@@ -293,92 +305,94 @@ public static void teardown() {
                     options.put("extraPlugin", true);
                     FinalCounts finalCounts = new FinalCounts(TEST_SIZE, TEST_SIZE + 1, TEST_SIZE + 1, 1, TEST_SIZE, 0, TEST_SIZE, 0, TEST_SIZE/BATCH_SIZE, 0, "FINISHED");
                     finalCounts.optionsFile = "options-extra";
-                    testHarmonizeFlow(prefix, codeFormat, dataFormat, options, stagingClient, HubConfig.DEFAULT_FINAL_NAME, finalCounts, true);
+                    testHarmonizeFlow(prefix, codeFormat, dataFormat, useEs, options, stagingClient, HubConfig.DEFAULT_FINAL_NAME, finalCounts, true);
                 }));
 
                 tests.add(DynamicTest.dynamicTest(flowName + " extra error", () -> {
                     Map<String, Object> options = new HashMap<>();
                     options.put("extraPlugin", true);
                     options.put("extraGoBoom", true);
                     FinalCounts finalCounts = new FinalCounts(TEST_SIZE, TEST_SIZE, TEST_SIZE + 1, 1, TEST_SIZE - 1, 1, TEST_SIZE - 1, 1, TEST_SIZE/BATCH_SIZE, 0, "FINISHED_WITH_ERRORS");
-                    testHarmonizeFlowWithFailedMain(prefix, codeFormat, dataFormat, options, stagingClient, HubConfig.DEFAULT_FINAL_NAME, finalCounts);
+                    testHarmonizeFlowWithFailedMain(prefix, codeFormat, dataFormat, useEs, options, stagingClient, HubConfig.DEFAULT_FINAL_NAME, finalCounts);
                 }));
             }
         });
 
-        allCombos((codeFormat, dataFormat, flowType) -> {
+        Path entityDir = projectDir.resolve("plugins").resolve("entities").resolve(ENTITY);
+
+        allCombos((codeFormat, dataFormat, flowType, useEs) -> {
             String prefix = "scaffolded";
-            String flowName = getFlowName(prefix, codeFormat, dataFormat, flowType);
-            if (flowType.equals(FlowType.INPUT)) {
+            String flowName = getFlowName(prefix, codeFormat, dataFormat, flowType, useEs);
+            if (flowType.equals(FlowType.INPUT) && useEs) {
```

Comment:
are there input flows that don't use ES? those would go to the else clause, which tests harmonize flows.

Reply:
good catch.